packages/@angular/cli/blueprints/ng2/files/__path__/tsconfig.json

```diff
@@ -17,5 +17,9 @@
     "typeRoots": [
       "<%= relativeRootPath %>/node_modules/@types"
     ]
-  }
+  },
+  "exclude": [
+    "test.ts",
```

Comment:
I think you won't be able to get some IDE support in tests if you exclude these.